source/stylesheets/refills/_device.scss

```diff
@@ -44,19 +28,14 @@
     }
 
     h4 {
-      border-bottom: 1px solid transparentize($device-text-color, 0.7);
-      margin-bottom: 0.5em;
-      padding-bottom: 0.5em;
+      font-size: 1.5em;
+      margin: 0 0 $small-spacing 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 0 $small-spacing` instead of `0 0 $small-spacing 0`